src/Compilers/Core/Portable/Optional.cs

```diff
@@ -56,5 +56,15 @@ public T Value
         {
             return new Optional<T>(value);
         }
+
+        /// <summary>
+        /// Returns a string representation of the optional value.
+        /// </summary>
+        public override string ToString()
+        {
+            return _hasValue
+                ? _value.ToString()
```

Comment:
> in case the value itself is null

Oh right, it's easy to forget how broken this implementation of `Optional<T>` is.  😨